### PR TITLE
[Waiting for Contributor to Sign CLA] Fixed wrong column of sub query

### PR DESCRIPTION
**Pull request by zhangever**
### What is changed, added or deleted? (Required)

修正文档中的错误的子查询列

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version) **To be manually cherry-picked**
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@zhangever Thanks for your contribution! Could you sign the [CLA](https://github.com/pingcap/docs-cn/pull/4561#issuecomment-697114289) for this PR to be merged?

**Review by winoros — COMMENTED** [review]
lgtm

File: sql-logical-optimization.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ aliases: ['/docs-cn/stable/sql-logical-optimization/','/docs-cn/v4.0/sql-logical
 
 # 逻辑优化
 
-本章节将对一些比较关键的逻辑改写进行说明，帮助大家理解 TiDB 如何生成最终的查询计划。比如在 TiDB 输入 `select * from t where t.a in (select * from t1 where t1.b=t.b)` 这个查询时，在最终的执行计划中将看不到这个 `t.a in (select t1.a from t1 where t1.b=t.b` 这个 `IN` 子查询的存在，这便是因为 TiDB 对这里进行了一些改写。
+本章节将对一些比较关键的逻辑改写进行说明，帮助大家理解 TiDB 如何生成最终的查询计划。比如在 TiDB 输入 `select * from t where t.a in (select t1.a from t1 where t1.b=t.b)` 这个查询时，在最终的执行计划中将看不到这个 `t.a in (select t1.a from t1 where t1.b=t.b` 这个 `IN` 子查询的存在，这便是因为 TiDB 对这里进行了一些改写。
```
Comment (lilin90) [inline sql-logical-optimization.md:8]: Perhaps missing a parenthesis? @winoros PTAL
```suggestion
本章节将对一些比较关键的逻辑改写进行说明，帮助大家理解 TiDB 如何生成最终的查询计划。比如在 TiDB 输入 `select * from t where t.a in (select t1.a from t1 where t1.b=t.b)` 这个查询时，在最终的执行计划中将看不到这个 `t.a in (select t1.a from t1 where t1.b=t.b)` 这个 `IN` 子查询的存在，这便是因为 TiDB 对这里进行了一些改写。
```

**Comment by lilin90** [comment]
@zhangever Would you please change the target branch from `release-4.0` to `master`? So the cherry-picker can automatically create a PR to the other branch.
image

**Comment by TomShawn** [comment]
> @zhangever Would you please change the target branch from `release-4.0` to `master`? So the cherry-picker can automatically create a PR to the other branch.
> image

I am afraid this PR has to be manually cherry-picked to `master` because its branch cannot be simply rebased on this page.

File: sql-logical-optimization.md (hunk 1 above)
Comment (TomShawn) [reply sql-logical-optimization.md:8]: According to @winoros, yes.

**Comment by TomShawn** [comment]
@zhangever Because it's been quite a while for the CLA to be signed, to better manage the PRs in this repo, I'll close this PR and migrate your accepted change to https://github.com/pingcap/docs-cn/pull/4771. If you would like to make more contribution to our documentation in the future, you are welcome to create a PR (recommended to pick one of the [issues](https://github.com/pingcap/docs-cn/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue)).

**Pull request closed without merging**